### ISSUE TITLE
It would be nice to apply @aws.cloudformation#cfnExcludeProperty  to simple shapes

### ISSUE THREAD

**Issue by prestomation** [smithy-lang/smithy#923]
cfnExcludeProperty  has a trait selector of `structure > member`.

It would be a nicer model in some cases if this could be applied to a simple shape, and have the exclusion apply to all uses of this shape. 

Take this API as an example

```
@aws.cloudformation#cfnExcludeProperty
string ClientToken

structure CreateResource {
    
    @idempotencyToken
    @aws.cloudformation#cfnExcludeProperty
    clientToken: ClientToken,

}

structure CreateADifferentResource {
    
    @idempotencyToken
    @aws.cloudformation#cfnExcludeProperty
    clientToken: ClientToken,

}
```

As it is today, I have to mark this exclusion on each request structure that has this idempotency token.

I know that I never want members of type ClientToken to be included in a CFN schema. Support for this trait on the simple shape would simplify the overal model, as well as prevent errors when a developer forgets to mark a new use of this type appropriately.

**Comment by mtdowling** [comment]
Would the mixins proposal address this in a different way? For example, you could define that same `clientToken` member once and then reuse it, along with the exclusion. Not to say we can't look into excluding shapes too though.

**Comment by syall** [comment]
Mixins are currently shipped in Smithy 2.0 and can be used to avoid duplication as a workaround, but stabilization of traits will be investigated in #1805.

**Comment by haydenbaker** [comment]
Closing this since we're tracking it as part of #1805
